types/jsonld/index.d.ts

```diff
@@ -0,0 +1,217 @@
+// Type definitions for jsonld 1.5
+// Project: https://github.com/digitalbazaar/jsonld.js
+// Definitions by: Jonas Erbe <https://github.com/jason076>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.6
+
+import { Context, JsonLd, Document, Url, JsonLdProcessor, RemoteDocument } from './jsonld-spec';
+
+// Some typealiases for better readability and some placeholders
+type MimeNQuad = 'application/n-quads';
+type RdfDataSet = any;
+type RdfOrString = RdfDataSet|string;
+type Callback<T> = (err: Error, res: T) => void;
+type DocCallback = Callback<JsonLd>;
+
+/*
+ * Declares interfaces used to type the methods options object.
+ * The interfaces are usefull to avoid code replication.
+ */
+
+declare namespace Options {
```

Comment:
If these `Options` types are not also exported, consumers will have a hard time using the options types themselves if they want to reuse data. It also makes it impossible to modify the options types via module augmentations if a consumer finds something is missing and wants to test the change locally.

Reply:
You were right. I have exported them.